region/lowernorfair/east/Wasteland.json

```diff
@@ -238,7 +238,10 @@
           "length": 8,
           "openEnd": 0
         }
```

Comment:
Should these Wasteland Kzan Runways include `canEnemyExtendRunway`? (on [1, 1] and [5, 1])

Reply:
Clipping into the spikes perhaps, but running on them is intuitive already.